developer_tests/window/work/threed_model_mod.f90

```diff
@@ -0,0 +1,298 @@
+! DART software - Copyright UCAR. This open source software is provided
+! by UCAR, "as is", without charge, subject to all terms of use at
+! http://www.image.ucar.edu/DAReS/DART/DART_download
+!
+
+module model_mod
+
+! This is a template showing the interfaces required for a model to be compliant
+! with the DART data assimilation infrastructure. Do not change the arguments
+! for the public routines.
+
+use        types_mod, only : r8, i8, MISSING_R8
+
+use time_manager_mod, only : time_type, set_time
+
+use     location_mod, only : location_type, get_close_type, &
+                             loc_get_close_obs => get_close_obs, &
+                             loc_get_close_state => get_close_state, &
+                             set_location, set_location_missing
+
+use    utilities_mod, only : error_handler, &
+                             E_ERR, E_MSG, &
+                             nmlfileunit, do_output, do_nml_file, do_nml_term,  &
+                             find_namelist_in_file, check_namelist_read
+
+use netcdf_utilities_mod, only : nc_add_global_attribute, nc_synchronize_file, &
+                                 nc_add_global_creation_time, &
+                                 nc_begin_define_mode, nc_end_define_mode
+
+use state_structure_mod, only : add_domain, get_domain_size
+
+use ensemble_manager_mod, only : ensemble_type
+
+! These routines are passed through from default_model_mod.
+! To write model specific versions of these routines
+! remove the routine from this use statement and add your code to
+! this the file.
```

Comment:
remove this comment?

Reply:
This is the template model_mod, it it just in there to compile the tests.

Reply:
I can change the quickbuild.sh to get the template model mod rather than a copying (lazy see commit https://github.com/NCAR/DART/commit/9d0f897b0827fcfef8ecd564d03237eb27d14247)

Reply:
Doesn't matter if we aren't going to put the tests in the repo, but if we do I think we should use the template

Reply:
used template, added fortran-test anything to developer tests.